src/config.c

```diff
@@ -20,28 +20,51 @@
 #include "resp3.h"
 #include "util/workers.h"
 
-#include "util/config_macros.h"
-
 #define __STRINGIFY(x) #x
 #define STRINGIFY(x) __STRINGIFY(x)
 
 #define RS_MAX_CONFIG_TRIGGERS 1 // Increase this if you need more triggers
 RSConfigExternalTrigger RSGlobalConfigTriggers[RS_MAX_CONFIG_TRIGGERS];
 
+int set_numeric_config(const char *name, long long val, void *privdata,
+                  RedisModuleString **err) {
+  *(long long *)privdata = val;
+  return REDISMODULE_OK;
+}
+
+long long get_numeric_config(const char *name, void *privdata) {
+  return (*(long long *)privdata);
+}
+
 // EXTLOAD
 CONFIG_SETTER(setExtLoad) {
   int acrc = AC_GetString(ac, &config->extLoad, NULL, 0);
   RETURN_STATUS(acrc);
 }
 
 CONFIG_GETTER(getExtLoad) {
-  if (config->extLoad) {
+  if (config->extLoad && strlen(config->extLoad) > 0) {
     return sdsnew(config->extLoad);
   } else {
     return NULL;
   }
 }
 
+// ext-load
+CONFIG_API_STRING_SETTER(set_ext_load);
+
+RedisModuleString* get_ext_load(const char *name, void *privdata) {
+  char *str = *(char **)privdata;
+  if (str) {
```

Comment:
Can avoid this check? (`privdata` is initialized properly)

Reply:
No, we can't. `RSGlobalConfig.extLoad = NULL` by default.
CONFIG GET returns `""` when the variable is `NULL`

```sh
127.0.0.1:6379> ft.config get extload
1) 1) EXTLOAD
   2) (nil)
127.0.0.1:6379> config get search-ext-load
1) "search-ext-load"
2) ""
```